### PR TITLE
[WIP] [14.0] [MIG] document_url

### PR DESCRIPTION
**Pull request by DavidBForgeFlow**
Standard migration to v14.

### CODE REVIEW

File: document_url/static/src/js/url.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ */
+odoo.define("document_url", function (require) {
+    "use strict";
+
+    var AttachmentBox = require("mail.AttachmentBox");
+
+    AttachmentBox.include({
+        events: _.extend(AttachmentBox.prototype.events, {
+            "click button.o_add_url_button": "_onAddUrl",
```
Comment (MiquelRForgeFlow) [inline document_url/static/src/js/url.js:15]:
```suggestion
            "click span.o_add_url_button": "_onAddUrl",
```

**Comment by CasVissers-360ERP** [comment]
@DavidBForgeFlow Tried it on runbot but not working, WIP? 
https://drive.google.com/file/d/1KG2F5Veaqok3EwoHBGGJavGzq6F2rlz0/view

File: document_url/static/src/xml/url.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+                type="button"
+                class="o_add_url_button btn btn-link"
+                t-on-click="_onClickAdd"
+            >
+                    <i class="fa fa-plus-square" />
+                    Add URL
+            </button>
+        </t>
+    </t>
+</templates>
```
Comment (victoralmau) [inline document_url/static/src/xml/url.xml:17]:
```suggestion
<templates xml:space="preserve">
    <t t-inherit="mail.AttachmentBox" t-inherit-mode="extension" owl="1">
        <xpath expr="//button[hasclass('o_AttachmentBox_buttonAdd')]" position="after">
            <button
                type="button"
                class="o_add_url_button btn btn-link"
                t-on-click="_onAddUrl"
            >
                    <i class="fa fa-plus-square" />
                    Add URL
            </button>
        </xpath>
    </t>
</templates>
```
I think this change is necessary to show the "Add URL" button.

**Review by victoralmau — CHANGES_REQUESTED** [review]
In addition, it will be necessary to change the javascript to adapt it to v14 and to extend the `AttachmentBox` component so that it works correctly.

**Comment by pedrobaeza** [comment]
I think you can supersede, @victoralmau, as the contributor seems unresponsive.

**Comment by MiquelRForgeFlow** [comment]
This PR should be closed, as @DavidBForgeFlow is no longer active and the PR was clearly superseded.

**Pull request closed without merging**